src/components/sections/general/form-sections/sync-input-phone-field.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from '@dhis2/prop-types'
+import { InputField } from '@dhis2/ui'
+import { FormSection } from './form-section'
+
+export const SyncInputPhoneField = ({ handleForm, syncElement }) => {
+    const { label, helpText, syncType } = syncElement
+    return (
+        <FormSection>
+            <InputField
+                type="tel"
+                inputWidth="250px"
+                helpText={i18n.t('{{helpText}}', { helpText })}
+                label={i18n.t('{{label}}', { label })}
```

Comment:
Calling `i18n.t` here with only a variable doesn't do anything, the text needs to be translated before it is passed to this component.